src/passes/OptimizeInstructions.cpp

```diff
@@ -570,6 +570,22 @@ struct OptimizeInstructions
         return ret;
       }
     } else if (auto* unary = curr->dynCast<Unary>()) {
+      if (unary->op == EqZInt32 || unary->op == EqZInt64) {
+        using namespace Match;
+        Const* c;
+        Binary* inner;
+        Expression* left;
```

Comment:
It looks like `left` is only used to get the child type of the binary expression. You should be able to save a variable by using `c->type` for that instead.

Reply:
Done!